airlock_processor/StatusChangedQueueTrigger/__init__.py

```diff
@@ -28,50 +30,46 @@ def __init__(self, source_account_name: str, dest_account_name: str):
         self.dest_account_name = dest_account_name
 
 
-def main(msg: func.ServiceBusMessage, outputEvent: func.Out[func.EventGridOutputEvent]):
+def main(msg: func.ServiceBusMessage, stepResultEvent: func.Out[func.EventGridOutputEvent], toDeleteEvent: func.Out[func.EventGridOutputEvent]):
```

Comment:
```suggestion
def main(msg: func.ServiceBusMessage, stepResultEvent: func.Out[func.EventGridOutputEvent], containerDeletionEvent: func.Out[func.EventGridOutputEvent]):
```

Reply:
changed to dataDeletionEvent as discussed below